visualization/app/codeCharta/ui/metricDeltaSelected/metricDeltaSelected.component.ts

```diff
@@ -35,7 +35,7 @@ export class MetricDeltaSelectedController implements BuildingSelectedEventSubsc
 		}, MetricDeltaSelectedController.TIME_TO_INIT_BINDING)
 	}
 
-	public onBuildingSelected(selectedBuilding: CodeMapBuilding) {
+	public onBuildingSelected(selectedBuilding?: CodeMapBuilding) {
```

Comment:
there exists an interface of this. Let's update the parameter to optional there as well?